lib/base-compiler.js

```diff
@@ -177,6 +177,11 @@ Compile.prototype.generateAST = function (inputFilename, options) {
         .then(this.processAstOutput);
 };
 
+Compile.prototype.getOutputFilename = function (dirPath, outputFilebase) {
```

Comment:
You probably can get away by not using a temporary here :)

Reply:
I'm not sure what you mean by that, can you expand on that?